.github/workflows/unit-manual.yml

```diff
@@ -0,0 +1,67 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Manual Unit Tests
+on:
+  workflow_dispatch:
+    inputs:
+      profile:
+        description: 'Profile'
+        required: false
+        default: 'allTests'
+        type: choice
+        options:
+          - allTests
+          - ee-namespace-interop
+      os:
+        description: 'OS'
+        required: false
+        default: 'ubuntu-latest'
+        type: choice
+        options:
+          - ubuntu-latest
+          - windows-latest
+          - macos-latest
+      jdk-distribution:
+        description: 'JDK Distribution'
+        required: false
+        default: 'temurin'
+        type: choice
+        options:
+          - temurin
+          - semeru
+          - microsoft
+          - oracle
+          - zulu
+          - corretto
+          - liberica
+      jdk-version:
+        description: 'JDK Version'
+        required: false
+        default: '11'
+        type: choice
+        options:
+          - 11
+          - 17
+
+jobs:
+  build-test:
+    name: ${{ inputs.jdk-distribution}}-${{ inputs.jdk-version }}-${{ inputs.os }}
+    runs-on: ${{ inputs.os }}
+    timeout-minutes: 25
+    strategy:
+      fail-fast: false
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up JDK ${{ inputs.jdk-distribution }} ${{ inputs.jdk-version }}
+      uses: actions/setup-java@v3
+      with:
+        distribution: ${{ inputs.jdk-distribution }}
+        java-version: ${{ inputs.jdk-version }}
+    - name: Run Tests
+      run: mvn -ntp -U -B -fae -P${{ inputs.profile }} clean verify
```

Comment:
BTW You may consider caching the local maven repository - e.g. https://github.com/modcluster/mod_cluster/blob/main/.github/workflows/ci.yml#L25-L31

Reply:
@rhusar thanks for the tip. We'll consider using cache in future improvement.